File: lab02/lab02/Item.cs
namespace lab02;

public class Item(int number, int value, int weight)
{
    public int Number { get; private set; } = number;
    public int Value { get; private set; } = value;
    public int Weight { get; private set; } = weight;
    public double Ratio() => (double)Value / Weight;

    public override string ToString()
    {
        return $"Item(n={Number}, value={Value}, weight={Weight})";
    }
}

File: lab06/Knapsack/src/main/java/com/raczu/knapsack/Item.java
package com.raczu.knapsack;

public class Item {
    public int number;
    public int value;
    public int weight;

    public Item(int number, int value, int weight) {
        this.number = number;
        this.value = value;
        this.weight = weight;
    }

    public float ratio() {
        return (float) this.value / this.weight;
    }

    @Override
    public String toString() {
        return String.format("Item(n=%d, value=%d, weight=%d)", this.number, this.value, this.weight);
    }
}
